credoai/modules/dataset_modules/dataset_base.py

```diff
@@ -24,8 +24,10 @@ def run(self):
         pipe = self._make_pipe()
         cv_results = self._run_cv(pipe)
         group_differences = self._group_differences()
+        feature_space_proxy_scores = self._assess_proxy_in_feature_space()
         self.results = {'sensitive_feature_roc_auc': cv_results.mean(),
```

Comment:
```suggestion
        self.results = {'overall_proxy_score': cv_results.mean(),
```

Renaming to bring in line with your `feature_space_proxy_scores`

Reply:
I change this to mutual information for accuracy.

Reply:
This hasn't been changed yet